osu.Game/Overlays/MusicController.cs

```diff
@@ -279,6 +279,10 @@ private void restartTrack()
 
         private void changeBeatmap(WorkingBeatmap newWorking)
         {
+            // If the provided beatmap is same as current, then there is no need to do any changes.
```

Comment:
Rather than a comment that matches the code on the line below (100% useless), this should explain *how* this can happen, ie. under what scenarios the event is fired twice (and why).

This is important because it's done this way for a reason and may be accidentally changed in the future if someone isn't aware of this reasoning.

It probably should have already been commented in the calls to `changeBeatmap`, but you can find the reasoning here as it may not be immediately obvious (bindable execution order): https://ptb.discordapp.com/channels/188630481301012481/188630652340404224/746293044302577684

Reply:
Something like:

```
This method can potentially be triggered multiple times as it is eagerly fired in next() / prev() to ensure correct execution order (changeBeatmap must be called before consumers receive the bindable changed event, which is not the case when called on the local bindable itself).
```